packages/react/src/elements/components/Flyout.js

```diff
@@ -50,8 +61,16 @@ Flyout.propTypes = {
    * Content for the flyout
    */
   content: PropTypes.node,
+  /**
+   * When true, renders the flyout open on mount
+   */
+  initiallyOpen: PropTypes.bool,
```

Comment:
What do you think about an interface like this instead?

```
open: PropTypes.bool, // <- When this is provided it overrides state
onClickOutside: PropTypes.func // <- pass this through to the vanilla component
```

Reply:
Sure, I can make it happen. Anna mentioned there are other components whose props override state. I'll find an example.

Reply:
It ends up looking like this a lot of the time:

https://github.com/Autodesk/hig/blob/8f82ff075cf2e055cd54366c4d400b78c4d9c640/packages/react/src/adapters/HIGAdapter/ControlsProp.js#L79